.github/workflows/build.yml

```diff
@@ -36,7 +32,7 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v1
         with:
-          node-version: 12.x
+          node-version: 14.x
```

Comment:
Is there anything specifically preventing us from using 16?

Reply:
I think this task doesn't build in node 16. We could later try upgrading some dependencies and try harder